modules/osdk-hh-create-go-api.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-hh-create-go-api_{context}"]
+= Creating a Go API
+
+Use the Operator SDK CLI to create a Go API.
+
+.Procedure
+
+. Run the following command to create a new custom resource definition (CRD) API with group `cache`, version `v1`, and kind `MemcachedBackup`:
```

Comment:
The intro sentence for Creating a Helm API states: _Run the following command to create a Helm API _. Should the two use the same structure? _Run the following command to create a Go API_.